src/parser/core/Events.ts

```diff
@@ -163,9 +163,17 @@ export type AnyEvent<
 > = MappedEventTypes[ET];
 
 export interface Ability {
+  /** The ability's name */
   name: string;
+  /**
+   * The ability's ID - this uniquely identifies the ability,
```

Comment:
This is literally the most minor thing event but if we are adding comments we should probably mention that
A single spell can have multiple GUIDs
A single GUID can handle multiple parts of a spell
And that this alone might not be all the information you need about a spell

Reply:
Yeah I can add to it. I added these comments just because I was trying to figure out what `ability.type` was and so after hunting down what it was I decided to comment the whole object.